.github/workflows/buildService.yml

```diff
@@ -33,9 +27,11 @@ jobs:
           make
           PACKAGE_ID=$(yq -oy ".id" manifest.*)
           echo "package_id=$PACKAGE_ID" >> $GITHUB_ENV
+          printf "\n SHA256SUM: $(sha256sum ${PACKAGE_ID}.s9pk) \n"
         shell: bash
+
       - name: Upload .s9pk
         uses: actions/upload-artifact@v4
         with:
           name: ${{ env.package_id }}.s9pk
-          path: ./${{ env.package_id }}.s9pk
+          path: ./${{ env.package_id }}.s9pk
```

Comment:
unintentional?

Reply:
This like the deletion of the trailing whitespace flagged. This should be fine though.